modules/github.js

```diff
@@ -6,33 +6,53 @@ const { createNetrcAuth } = require("octokit-auth-netrc");
 const echo = require("./echo");
 const fs = require("fs");
 const git = require("./git");
+const js = require("./js");
 const { Octokit } = require("@octokit/rest");
 const os = require("os");
 const upath = require("upath");
 const userPrompt = require("./user-prompt");
+const { StringUtils } = require("andculturecode-javascript-core");
+const { fork } = require("child_process");
+const { sleep } = require("./js");
```

Comment:
Nitpick - in terms of alphabetization, I've been trying to keep the destructured imports above the standard module imports. What do you think?

Reply:
Yeah, I'm down for that. Definitely helps grouping them